app/src/main/res/layout-sw320dp/activity_description.xml

```diff
@@ -73,14 +74,14 @@
                                 android:layout_width="wrap_content"
                                 android:layout_height="wrap_content"
                                 android:fontFamily="@font/armata"
-                                android:text="By " />
+                                android:text="@string/activity_description_by" />
 
                             <TextView
                                 android:id="@+id/activity_organisator_description"
                                 android:layout_width="wrap_content"
                                 android:layout_height="wrap_content"
                                 android:fontFamily="@font/armata"
-                                android:text="Organisator" />
+                                android:text="@string/activity_description_Organisator" />
```

Comment:
"Organizer" instead of "Organisator" sounds more english ;)

Reply:
Haha thank you, It's true that I took back what was written so I will quickly change that, it will be less French !

Reply:
It was my mistake, sorry